db/schema.rb

```diff
@@ -279,26 +282,26 @@
     t.text "application_purpose"
     t.boolean "proceedings_before_the_court"
     t.text "details_of_proceedings_before_the_court"
-    t.decimal "estimated_legal_cost", precision: 10, scale: 2
     t.string "success_prospect"
     t.text "success_prospect_details"
+    t.decimal "estimated_legal_cost", precision: 10, scale: 2
     t.datetime "submitted_at"
     t.index ["legal_aid_application_id"], name: "index_merits_assessments_on_legal_aid_application_id"
   end
 
   create_table "other_assets_declarations", id: :uuid, default: -> { "gen_random_uuid()" }, force: :cascade do |t|
     t.uuid "legal_aid_application_id", null: false
-    t.decimal "second_home_value", precision: 14, scale: 2
-    t.decimal "second_home_mortgage", precision: 14, scale: 2
-    t.decimal "second_home_percentage", precision: 14, scale: 2
-    t.decimal "timeshare_value", precision: 14, scale: 2
-    t.decimal "land_value", precision: 14, scale: 2
-    t.decimal "jewellery_value", precision: 14, scale: 2
-    t.decimal "vehicle_value", precision: 14, scale: 2
-    t.decimal "classic_car_value", precision: 14, scale: 2
-    t.decimal "money_assets_value", precision: 14, scale: 2
-    t.decimal "money_owed_value", precision: 14, scale: 2
-    t.decimal "trust_value", precision: 14, scale: 2
+    t.decimal "second_home_value"
+    t.decimal "second_home_mortgage"
+    t.decimal "second_home_percentage"
+    t.decimal "timeshare_value"
+    t.decimal "land_value"
+    t.decimal "jewellery_value"
+    t.decimal "vehicle_value"
+    t.decimal "classic_car_value"
+    t.decimal "money_assets_value"
+    t.decimal "money_owed_value"
+    t.decimal "trust_value"
     t.datetime "created_at", null: false
```

Comment:
How did the precision come to be removed from these fields in the schema?

Reply:
This file contains lots of unnecessary change. It happens when migrations are not run in the same order.
@mpw5 , if you want to reset your local DB to match `db/schema.rb`, you can run `bin/rails db:schema:load db:seed` (it will remove all data). And then re-run your migration

Reply:
Thanks both. Fixed now.